regression/cbmc/r_w_ok7/main.c

```diff
@@ -0,0 +1,17 @@
+#include <assert.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+int main()
+{
+  size_t x;
+  size_t y;
+  uint8_t *a;
+
+  __CPROVER_assume(x > 0);
+  __CPROVER_assume(y > x);
+
+  a = malloc(sizeof(*(a)) * (x));
+
+  assert(!__CPROVER_w_ok(a, y));
```

Comment:
:no_entry_sign: I think you should add 
```suggestion
  assert(!__CPROVER_w_ok(a, y));
  assert(__CPROVER_w_ok(a, x));
```
to check cprover_w_ok with a dynamic offset works